test/fixtures/api-client/location.find.json

```diff
@@ -17,7 +17,11 @@
           "created_at": "2019-10-09T11:10:53+01:00",
           "updated_at": "2019-10-09T11:10:53+01:00"
         }
-      ]
+      ],
+      "category": {
```

Comment:
The categories in both this, and the `location.findAll.json` are synthetic, and are enough to pass the tests. 

Do they need to be more authentically recorded from staging?

Reply:
I don't think so

Reply:
If we decide later that they should be, we can re-record from staging.